### DISCUSSION TITLE
Authelia : Home>Configuration>First Factor>LDAP expecting step by step to set certificate chain

### DISCUSSION THREAD

**Discussion by Nomtest** [authelia/authelia#4301] [Q&A] [answered]
Hello,

Is it possible to get a step by step to know how to manage {certificate_chain} value in the authentication_backend part of the configuration ? (with /bin/openssl I think...)

tnx from no IT pro to get authelia enable faster than studying all the IT... :-)

**Comment by james-d-elliott** [comment, marked as answer]
I would strongly recommend instead of asking about your intended solution that you ask about your particular problem. However all of the TLS settings are described at a link in the article: https://www.authelia.com/configuration/first-factor/ldap/#tls

**Comment by Nomtest** [comment]
well the point is to get a good bind between Authelia and OpenLDAP provider :

But : 

time="2022-10-31T11:58:19+01:00" level=info msg="Storage schema is being checked for updates"
time="2022-10-31T11:58:20+01:00" level=info msg="Storage schema is already up to date"
time="2022-10-31T11:58:20+01:00" level=error msg="Failure running the user provider startup check: bind failed with error: unable to read LDAP response packet: unexpected EOF" stack="github.com/authelia/authelia/v4/internal/commands/root.go:188 doStartupChecks\ngithub.com/authelia/authelia/v4/internal/commands/root.go:82  cmdRootRun\ngithub.com/spf13/cobra@v1.5.0/command.go:876                  (*Command).execute\ngithub.com/spf13/cobra@v1.5.0/command.go:990                  (*Command).ExecuteC\ngithub.com/spf13/cobra@v1.5.0/command.go:918                  (*Command).Execute\ngithub.com/authelia/authelia/v4/cmd/authelia/main.go:10       main\nruntime/proc.go:250                                           main\nruntime/asm_amd64.s:1571                                      goexit"
time="2022-10-31T11:58:20+01:00" level=fatal msg="The following providers had fatal failures during startup: user" stack="github.com/authelia/authelia/v4/internal/commands/root.go:210 doStartupChecks\ngithub.com/authelia/authelia/v4/internal/commands/root.go:82  cmdRootRun\ngithub.com/spf13/cobra@v1.5.0/command.go:876                  (*Command).execute\ngithub.com/spf13/cobra@v1.5.0/command.go:990                  (*Command).ExecuteC\ngithub.com/spf13/cobra@v1.5.0/command.go:918                  (*Command).Execute\ngithub.com/authelia/authelia/v4/cmd/authelia/main.go:10       main\nruntime/proc.go:250                                           main\nruntime/asm_amd64.s:1571                                      goexit"

And in PhpLdapAdmin, 
there is : 
      "This base cannot be created with PLA."   when connected with the ldap Authelia account.


=> how to set up the authelia user account in OpenLDAP ?

**Reply by james-d-elliott** [reply]
> well the point is to get a good bind between Authelia and OpenLDAP provider

Provide the LDAP configuration section, OpenLDAP logs, and information on the specifics of how the LDAP server is configured. What ports does it listen on? What transport method does it use LDAPS/StartTLS/Plain LDAP?

> time="2022-10-31T11:58:20+01:00" level=error msg="Failure running the user provider startup check: bind failed with error: unable to read LDAP response packet: unexpected EOF" stack="github.com/authelia/authelia/v4/internal/commands/root.go:188 doStartupChecks\ngithub.com/authelia/authelia/v4/internal/commands/root.go:82 cmdRootRun\ngithub.com/spf13/cobra@v1.5.0/command.go:876 (*Command).execute\ngithub.com/spf13/cobra@v1.5.0/command.go:990 (*Command).ExecuteC\ngithub.com/spf13/cobra@v1.5.0/command.go:918 (*Command).Execute\ngithub.com/authelia/authelia/v4/cmd/authelia/main.go:10 main\nruntime/proc.go:250 main\nruntime/asm_amd64.s:1571 goexit" 

This probably indicates your URL is misconfigured, or something related to it.

> => how to set up the authelia user account in OpenLDAP ?

This is not related to your error messages. You just need to create an account that has permissions to read attributes of all the users that you wish to be able to login, and if you want password reset functionality it needs permissions to write user passwords for the same group of users.

**Reply by Nomtest** [reply]
Ok, I will work with this : big tnx for your help, James-d-elliot.
